backend/cmd/api/main.go

```diff
@@ -225,6 +225,8 @@ func main() {
 			"x-api-key",
 			"x-application",
 			"x-feature-flags",
+			"x-session-id",
+			"x-search-session-id",
```

Comment:
Anything more than this needed to prevent CORS errors in frontend?

Reply:
Not afaik 👍